docs/10.EXT-REFERENCE-HANDLER.md

```diff
@@ -265,12 +265,13 @@ a backtrace is also printed out.
 
 ```c
 jerry_value_t
-jerryx_handler_assert_fatal (const jerry_value_t func_obj_val, const jerry_value_t this_p,
-                             const jerry_value_t args_p[], const jerry_length_t args_cnt);
+jerryx_handler_assert_fatal (const jerry_call_info_t *call_info_p,
```

Comment:
It looks like this has been renamed to `jerryx_handler_assert`. There are a few comments in the source code with the wrong name. Can you double check and fix those as well?